### ISSUE TITLE
windows vm compile takes 20m?

### ISSUE THREAD

**Issue by erg** [factor/factor#1451]
Compiling the vm takes forever. Why? (It only takes this long from the mason instance, maybe <1m from command-line?)

Also, we should add this compile time to the mason email.

```
C:\factor64>factor.com \Users\Doug\config\mason.factor
Mon, 17 Aug 2015 12:26:02 Building in directory C:\Users\Doug/builds/2015-08-17-12-26
Mon, 17 Aug 2015 12:26:02 Cloning GIT repository
Mon, 17 Aug 2015 12:27:27 Starting build of GIT ID 048bdca05049fbd6bde8db633422c9667ce84741
Mon, 17 Aug 2015 12:27:27 Compiling VM
Mon, 17 Aug 2015 12:48:46 Bootstrapping
Mon, 17 Aug 2015 13:01:29 Running tests
Mon, 17 Aug 2015 15:05:45 Build finished with status: status-clean
Mon, 17 Aug 2015 15:07:03 Uploaded factor-windows-x86-64-2015-08-17-12-26.zip
```

**Comment by mrjbq7** [comment]
Are you running in a VM?

It took me 30 minutes to unzip factor.zip

**Comment by erg** [comment]
It's my pretty decent windows computer and not a vm.

Intel i7-3820 @ 3.60 GHz



On Mon, Aug 17, 2015 at 3:20 PM, John Benediktsson <notifications@github.com

**Comment by mrjbq7** [comment]
So a child process build from within factor takes forever?  But from the command-line it works pretty fast?

**Comment by erg** [comment]
Yes

**Comment by mrjbq7** [comment]
weird!

**Comment by mrjbq7** [comment]
Be nice to fix this for 0.98

**Comment by erg** [comment]
It's fine now.

**Comment by erg** [comment]
Fixed in 37163a96d6071e98402e6782ca8917d735f925c6
